bundles/org.openhab.binding.linky/src/main/java/org/openhab/binding/linky/internal/handler/LinkyHandler.java

```diff
@@ -312,37 +341,58 @@ private void updateVAChannel(String channelId, double power) {
                 if (last.isAfter(endDay)) {
                     last = endDay;
                 }
-                report.addAll(reportValues(first, last, separator));
+                report.addAll(buildReport(first, last, separator));
                 first = last.plusDays(1);
             } while (!first.isAfter(endDay));
         }
         return report;
     }
 
     private @Nullable Consumption getConsumptionData(LocalDate from, LocalDate to) {
+        logger.debug("getConsumptionData from {} to {}", from.format(DateTimeFormatter.ISO_LOCAL_DATE),
+                to.format(DateTimeFormatter.ISO_LOCAL_DATE));
         EnedisHttpApi api = this.enedisApi;
         if (api != null) {
             try {
                 return api.getEnergyData(userId, prmId, from, to);
             } catch (LinkyException e) {
+                logger.debug("getConsumptionData failed: {}", e.getMessage(), e);
                 updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.OFFLINE.COMMUNICATION_ERROR, e.getMessage());
```

Comment:
It seems a bit unreasonable to log the stack trace when the network fails. Same for below.

Reply:
Changed

Reply:
Now we have a redundant logging as the message in `updateStatus()` is also logged by the framework. You could remove your logging entirely.

Reply:
It requires to enable an additional DEBUG log for a particular core bundle ?
My intentition was to get in an easy way feedback from users encountering problems; just by providing logs from the binding.
I even don't know myself what core bundle to enable log (without searching in the code), so imagine a standard user.
But the error will be available in MainUI, so we should request status from MainUI in case a user has a problem.

Reply:
Log removed

Reply:
This logging is enabled by default via [this ](https://github.com/openhab/openhab-distro/blob/master/distributions/openhab/src/main/resources/userdata/etc/log4j2.xml#L56) setting. It's `org.openhab.event.ThingStatusInfoChangedEvent` and logged to events.log.

Reply:
Ok thank you.